src/app/child-dev-project/children/children-list/children-list.component.ts

```diff
@@ -34,6 +34,8 @@ export class ChildrenListComponent implements OnInit {
 
   async ngOnInit() {
     this.route.data.subscribe(
+      // TODO replace this use of route and rely on the RoutedViewComponent instead
+      // see  that flattens the config option, assigning individual properties as inputs however, so we can't easily pass on
```

Comment:
I guess it will be quite some work to refactor this in the entity list component and then pass all the properties individually.

Reply:
I did start refactoring it but then realized it is quite complex. Also, we might prefer to move away from this ChildrenListComponent and use the EntityListComponent only (once we have those last few special bits generalized), so I didn't want to invest too much time for this now.